torch/ao/quantization/experimental/APoT_tensor.py

```diff
@@ -1,13 +1,62 @@
 import torch
 from torch import Tensor
+from torch.ao.quantization.experimental.observer import APoTObserver
+from torch.ao.quantization.experimental.apot_utils import float_to_apot, float_to_reduced_precision
 
 # class to store APoT quantized tensor
 class TensorAPoT(torch.Tensor):
-    @staticmethod
-    def quantize_APoT(tensor2quantize: Tensor) -> Tensor:
-        raise NotImplementedError
+    b: int
```

Comment:
this is not exactly following the C++ Tensor design actually, wondering if we want to fix it.
What we have in C++ is:
```
Quantizer class that has quantize and dequantize method and stores all the quantization parameters: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/quantized/Quantizer.h#L78

Quantized Tensor class that holds a quantizer object, stores integer representation in data, dtype will be quint8/qint8 etc.
int_repr is also a method on the quantized tensor, that will convert the quint8/qint8 tensor to uint8/int8 tensor and return the result
```

Reply:
yes, will fix